sdk/metricsadvisor/azure-ai-metricsadvisor/tests/async_tests/base_testcase_async.py

```diff
@@ -207,10 +207,10 @@ async def _create_data_feed(self, name):
             )
         )
 
-    async def _create_data_feed_and_anomaly_detection_config(self, name):
+    async def _create_data_feed_and_detection_config(self, name):
         data_feed = await self._create_data_feed(name)
-        detection_config_name = create_random_name(name)
-        detection_config = await self.admin_client.create_metric_anomaly_detection_configuration(
+        detection_config_name = self.create_random_name(name)
```

Comment:
this should not have the `self` for create_random_name. We're actually using a different one from the one described in AzureTestCase